homeassistant/components/swiss_public_transport/quality_scale.yaml

```diff
@@ -0,0 +1,86 @@
+rules:
+  # Bronze
+  action-setup: done
+  appropriate-polling:
+    status: done
+    comment: >
+      Polling interval is set to support one connection.
+      There is a rate limit at 10000 calls per day.
+  brands: done
+  common-modules: done
+  config-flow-test-coverage: done
+  config-flow: done
+  dependency-transparency: done
```

Comment:
From the guidelines: 

> The package published to PyPi should be built and published inside a CI pipeline.

That doesn't seem to be the case for this package.

Reply:
True, it is currently being pushed by hand. But the OG codeowner is not active anymore and I will look to take over if he is ok with it.

https://github.com/home-assistant-ecosystem/python-opendata-transport/tree/master/.github/workflows

Reply:
But this will happen in a separate PR